Orange/canvas/__main__.py

```diff
@@ -163,6 +169,9 @@ def main(argv=None):
 
     qt_argv += args
 
+    # Add --disable-web-security to enable Qt WebEngine to load local resources
+    qt_argv.append('--disable-web-security')
```

Comment:
@kernc Why is this necessary?

Qt WebEngine can (or at least should be able to) load local resources for locally originating content. At lest that is what the documentation for [QWebEnginePage](http://doc.qt.io/qt-5/qwebenginepage.html) says:
> ... By default, local schemes like file:// and qrc:// are considered to be in the same security origin, and can access each other's resources